src/routes/controllers/comments.controller.js

```diff
@@ -0,0 +1,84 @@
+import { errorResponse, successResponse } from '../../utils/helpers.utils';
+
+import models from '../../db/models';
+
+const { Comment, User, Article } = models;
+
+export const createComments = async (req, res) => {
+  const { id } = req.user;
+  const { articleId } = req.params;
+  const { comment } = req.body;
+
+  try {
+    const checkIfArticleExists = await Article.findByPk(articleId);
+    if (!checkIfArticleExists) {
+      return errorResponse(res, 404, 'This article does not exist');
+    }
+    const getUser = await User.findByPk(id);
+    if (!getUser) {
+      return errorResponse(res, 404, 'User does not exist');
+    }
+
+    const newComment = await Comment.create({
+      userId: id,
+      articleId,
+      body: comment,
+    });
+
+    if (newComment) {
```

Comment:
Am not sure there is any need for this `if` check here, you can just return newComment in the response.

Reply:
I have removed this if comment